### PR TITLE
Command documentation

### PR DESCRIPTION
**Pull request by arik123**
Added documentation for admin commands.
If you see some grammar mistakes let me know.

### CODE REVIEW

**Comment by Aethezxd** [comment]
133: Uploads a picture from link to your bots avatar

**Comment by Nicklason** [comment]
Make an object where all the commands are saved, search that object when using the help command. The objects should include the command, what it is used for, how it is used, and all the parameters / options it has.

File: app/messages.js [hunk 1]
```diff
@@ -59,9 +59,80 @@ function friendMessage (steamID, message) {
...
-            let reply = 'Here\'s a list of all my commands: !help, !message <message>, !how2trade, !stock, !price <name>, !buy <amount> <name>, !sell <amount> <name>';
-            if (Automatic.isOwner(steamID64)) {
-                reply += ', !add, !remove, !update, !profit, !removefriends, !use, !name, !avatar';
+            const helpCmd = message.substr(message.toLowerCase().indexOf('help') + 5).toLowerCase();
+            let reply = '';
+            if (helpCmd == '' || !Automatic.isOwner(steamID64)) {
+                reply = 'Here\'s a list of all my commands: !help, !message <message>, !how2trade, !stock, !price <name>, !buy <amount> <name>, !sell <amount> <name>';
+                if (Automatic.isOwner(steamID64)) {
+                    reply += '\n !add, !remove, !update, !profit, !removefriends, !use, !name, !avatar';
+                    reply += '\n for more informatio use !help <COMMAND>';
```
Comment (Gobot1234) [inline app/messages.js]: informatio

Comment (Gobot1234) [reply app/messages.js]: should be information

**Comment by Nicklason** [comment]
May include this in v2. But I think that help for the commands should be in the wiki or somewhere else.

**Pull request closed without merging**